README.md

```diff
@@ -15,8 +15,28 @@ The gulpfile is 80% from the [gulp-webapp](https://github.com/yeoman/generator-g
 
 There are two commands, both run via npm.
 
-- `npm run build` - clean & build everything and put it into dist folder
-- `npm run serve` - serve the pages and utilize live reload on changes to styles, fonts, images, scripts and HTML.
+- `npm run build` or `gulp build` or `gulp` - clean & build everything and put it into dist folder
```

Comment:
Huh, look at that. That seems very odd. I guess I'm still in favor of doing it via npm but now I have no basis for that opinion. :)